packages/web/src/components/data-entry/TagInput.js

```diff
@@ -38,9 +38,6 @@ class TagInput extends Component {
     if (this.state.typingMode && this.newTagInputRef.current) {
       this.newTagInputRef.current.focus()
     }
-    if (this.state.tags !== this.props.tags) {
```

Comment:
I assume this was here to allow for the component to be "controlled" externally. I imagine how we're using it is different now, but this might be unsafe?

Reply:
The only controlled usage is in the new upload form. I just added another round of changes that puts back the controlled behavior without breaking the old usages.

Also okay to just rip it out for now and revisit later.

Reply:
sweet! thanks for clarification